### PR TITLE
Updates for the IVR tutorial, adding next steps section

### PR DESCRIPTION
**Pull request by lornajane**
## Description

I did already update this tutorial once but I've got a better handle on what tutorials should have now so more updates! Sorry!
I'm aware of some issues with incoming links from nexmo.com/blog and have tickets open to have those updated to the building blocks instead of bits of tutorial.

### CODE REVIEW

**Comment by lornajane** [comment]
JIRA ticket for nexmo.com updates: https://nexmoinc.atlassian.net/browse/DOTCOM-1318

File: _tutorials/interactive-voice-response.md [hunk 1]
```diff
@@ -1,44 +1,43 @@
...
 
 # Interactive Voice Response
 
-Make it easy for visitors to navigate an Interactive Voice Response (IVR) application with simple text-to-speech (TTS) prompts.
+For users with a simple enquiry, make it easy and quick for them by offering an Interactive Voice Response (IVR) service. This tutorial will walk you through building an application to do exactly that with simple text-to-speech (TTS) prompts and keypad input.
 
-This tutorial is based on the [Simple IVR](https://www.nexmo.com/use-cases/interactive-voice-response/) use case. You download the code from <https://github.com/Nexmo/php-phone-menu>.
+The scenario is this: a customer phones a delivery company to find out the status of their order. They will be prompted to enter their order number and then hear a spoken response letting them know the (randomly generated by our example code) status of their order.
+
+This tutorial is based on the [Simple IVR](https://www.nexmo.com/use-cases/interactive-voice-response/) use case. All the code is available on GitHub: <https://github.com/Nexmo/php-phone-menu>.
```
Comment (marklewin) [inline _tutorials/interactive-voice-response.md]: Github link needs to be in markdown format.

File: _tutorials/interactive-voice-response.md [hunk 2]
```diff
@@ -1,44 +1,43 @@
...
 
 ## In this tutorial
 
-In this tutorial you build an interactive phone menu. The scenario is that a user is calling an automated service to get an update on the status of their order. We will set up a number to call, prompt a user to input their order with the keypad and then inform them of the (completely random and fictitious) status of their order.
+- [Get set up](#setting-up-for-ivr) - Create an application, configure it to point to your code, and set up the numbers you will use for this tutorial.
+- [Make the call](#make-the-call) - Make a phone call your application and walk through the prompts to hear the spoken information.
```
Comment (marklewin) [inline _tutorials/interactive-voice-response.md]: "Make a phone call your application" -> "Call your application"?

File: _tutorials/interactive-voice-response.md [hunk 3]
```diff
@@ -1,44 +1,43 @@
...
 
-- [Create a voice application](#create-a-voice-application) - create and configure an application using [Nexmo CLI](https://github.com/nexmo/nexmo-cli), then configure the webhook endpoints to provide NCCOs and handle changes in Call status
-- [Buy a phone number](#buy-a-phone-number) - buy voice enabled numbers for use in the application
-* [Link the phone numbers to the Nexmo Application](#link-phone-numbers-to-the-nexmo-application) - configure application to use the chosen numbers
-- [Try it yourself](#try-it-yourself) - enough reading the code, let's run it!
-- [Handle an inbound call](#handle-an-inbound-call) - configure your webhook endpoint to handle incoming voice calls, find the phone number it is associated with and handle the call
-- [Send a text-to-speech greeting](#send-text-to-speech-greeting) - Greet the user with text-to-speech upon answer
-- [Request user input via IVR](#request-user-input-via-ivr) - Create a text-to-speech prompt followed by requesting user input via IVR (Interactive Voice Response)
```
Comment (marklewin) [inline _tutorials/interactive-voice-response.md]: "Create a text-to-speech prompt followed by requesting..." -> "Create a text-to-speech prompt and then request"

File: _tutorials/interactive-voice-response.md [hunk 4]
```diff
@@ -1,44 +1,43 @@
...
+## Setting Up for IVR
 
 In order to work through this tutorial you need:
 
-* A [Nexmo account](https://dashboard.nexmo.com/sign-up)
-* The [Nexmo CLI](https://github.com/nexmo/nexmo-cli) installed and set up
-* A publicly accessible web server so Nexmo can make webhook requests to your app
-* The tutorial code from <https://github.com/Nexmo/php-phone-menu>
-
-If you're developing behind a firewall or a NAT, use [ngrok](https://ngrok.com/) to tunnel access to your Web server.
```
Comment (marklewin) [inline _tutorials/interactive-voice-response.md]: Could we perhaps link to an `ngrok` tutorial: https://www.nexmo.com/blog/2017/07/04/local-development-nexmo-ngrok-tunnel-dr/ ?

File: _tutorials/interactive-voice-response.md [hunk 5]
```diff
@@ -1,44 +1,43 @@
...
+* A [Nexmo account](https://dashboard.nexmo.com/sign-up).
+* The [Nexmo CLI](https://github.com/nexmo/nexmo-cli) installed and set up.
+* A publicly accessible PHP web server so Nexmo can make webhook requests to your app, or for local development we recommend [ngrok](https://ngrok.com).
```
Comment (marklewin) [inline _tutorials/interactive-voice-response.md]: Could we perhaps link to an ngrok tutorial: https://www.nexmo.com/blog/2017/07/04/local-development-nexmo-ngrok-tunnel-dr/ ?

File: _tutorials/interactive-voice-response.md [hunk 6]
```diff
@@ -1,44 +1,43 @@
...
+* The tutorial code from <https://github.com/Nexmo/php-phone-menu>, either clone the repository or download and extract the zip file to your machine.
```
Comment (marklewin) [inline _tutorials/interactive-voice-response.md]: Github link needs to be in markdown format.

File: _tutorials/interactive-voice-response.md [hunk 7]
```diff
@@ -102,7 +104,7 @@ php -S 0:8080 ./public/index.php
 
 Once it's running, call your nexmo voice number and follow the instructions! The code receives webhooks to `/event` as the call is started, ringing, etc. When the system answers the call, a webhook comes in to `/answer` and the code responds with some text-to-speech and then waits for user input. The user's input then arrives by webhook to '/search' and again the code responds with some text-to-speech.
```
Comment (marklewin) [inline _tutorials/interactive-voice-response.md]: '/search' needs to be surrounded by backticks: `/search`

File: _tutorials/interactive-voice-response.md [hunk 8]
```diff
@@ -285,11 +287,15 @@ As you can see from the search action, the sample application sends some rather
 Once the order information is passed to the user, they are told that they can hang up at anytime. The method that adds the order prompt NCCO is reused. That way the user can search for another order, but does not hear the welcome prompt every time.
 
 
-### Some finer details of the tutorial application
+## Tips for better Text-To-Speech experiences
 
-There are few more methods in our `Menu` code. Many times a phone menu interfaces with an existing system having data objects not easy converted to spoken prompts. In this example we have three things: objects that represent dates, an order ID, and a set of status constants.
+The `Menu` class has a few more methods that are there to improve the process of turning application data in to spoken prompts. Examples in this application include:
```
Comment (marklewin) [inline _tutorials/interactive-voice-response.md]: "in to" -> "into"

File: _tutorials/interactive-voice-response.md [hunk 9]
```diff
@@ -285,11 +287,15 @@ As you can see from the search action, the sample application sends some rather
...
 
-Properly communicating those values to the caller uses three methods. The `talkDate` method just returns a string with a date format that works well for spoken words.
+* the date of when the status was last reported
```
Comment (marklewin) [inline _tutorials/interactive-voice-response.md]: "date of when" -> "date when"

File: _tutorials/interactive-voice-response.md (hunk 5 above)
Comment (lornajane) [reply _tutorials/interactive-voice-response.md]: Or this https://developer.nexmo.com/concepts/guides/webhooks#using-ngrok-for-local-development

**Pull request merged**